### ISSUE TITLE
GStreamer benchmark sample gave "Permission denied"

### ISSUE THREAD

**Issue by alexlamfromhome** [dlstreamer/dlstreamer#325]
Although the sample run, logic that sets ie-config are bypassed due to the "Permission denied".
The script uses undefined variable PROCESSES_COUNT on line 49.

Running the benchmark
```
/opt/intel/dlstreamer/samples/gstreamer/benchmark$ ./benchmark_one_model.sh /mnt/nfs_Videos/cars-on-highway.1280x720.1min.mp4 ~/intel/dl_streamer/models/intel/person-vehicle-bike-detection-2004/FP16/person-vehicle-bike-detection-2004.xml CPU CPU 2 2
./benchmark_one_model.sh: line 49: 1: Permission denied
Setting pipeline to PAUSED ...
Setting pipeline to PAUSED ...
Setting pipeline to PAUSED ...
Pipeline is PREROLLING ...
Pipeline is PREROLLING ...
Redistribute latency...
Pipeline is PREROLLING ...
Redistribute latency...
Redistribute latency...
Redistribute latency...
Pipeline is PREROLLED ...
Setting pipeline to PLAYING ...
New clock: GstSystemClock
Pipeline is PREROLLED ...
Setting pipeline to PLAYING ...
New clock: GstSystemClock
Redistribute latency...
Redistribute latency...
Redistribute latency...0.0 %)
Redistribute latency...
FpsCounter(last 1.00sec): total=339.85 fps, number-streams=2, per-stream=169.93 fps (168.43, 171.42)
FpsCounter(average 1.00sec): total=339.84 fps, number-streams=2, per-stream=169.92 fps (168.43, 171.42)
FpsCounter(last 1.00sec): total=333.65 fps, number-streams=2, per-stream=166.83 fps (165.83, 167.82)
FpsCounter(average 2.00sec): total=336.75 fps, number-streams=2, per-stream=168.37 fps (167.13, 169.62)
FpsCounter(last 1.01sec): total=351.68 fps, number-streams=2, per-stream=175.84 fps (174.85, 176.83)
FpsCounter(average 3.01sec): total=341.73 fps, number-streams=2, per-stream=170.87 fps (169.70, 172.03)
```

Output from tops does not include ```ie-config```
```
 ps aux | grep gst-launch-1.0
alexlam   223759  839  2.4 3964324 388780 pts/0  Sl+  12:41   0:41 gst-launch-1.0 filesrc location=/mnt/nfs_Videos/cars-on-highway.1280x720.1min.mp4 ! decodebin force-sw-decoders=true ! video/x-raw ! gvainference model-instance-id=inf0 model=/home/alexlam/intel/dl_streamer/models/intel/person-vehicle-bike-detection-2004/FP16/person-vehicle-bike-detection-2004.xml device=CPU ! queue ! gvafpscounter write-pipe=/tmp/fps ! fakesink async=false
alexlam   223760  853  2.4 3964324 387816 pts/0  Sl+  12:41   0:42 gst-launch-1.0 filesrc location=/mnt/nfs_Videos/cars-on-highway.1280x720.1min.mp4 ! decodebin force-sw-decoders=true ! video/x-raw ! gvainference model-instance-id=inf0 model=/home/alexlam/intel/dl_streamer/models/intel/person-vehicle-bike-detection-2004/FP16/person-vehicle-bike-detection-2004.xml device=CPU ! queue ! gvafpscounter write-pipe=/tmp/fps ! fakesink async=false
alexlam   223761  0.6  0.1 197404 19120 pts/0    Sl+  12:41   0:00 gst-launch-1.0 gvafpscounter read-pipe=/tmp/fps interval=1 ! fakesink
alexlam   223861  0.0  0.0   9044   656 pts/1    S+   12:41   0:00 grep --color=auto gst-launch-1.0
```

**Comment by brmarkus** [comment]
Which exact version of DL-Streamer are you using, HEAD/latest, 2022.2, or 2022.2-pre-release?

Have you already tried replacing `PROCESSES_COUNT` with `NUMBER_PROCESSES`?

**Comment by mikhail-nikolskiy** [comment]
Please try benchmark samples from latest master branch, hot-fix commit today should fix the problem.

**Comment by pk1d3v** [comment]
Fixed in [2022.3](https://github.com/dlstreamer/dlstreamer/releases/tag/2022.3-release)
